Prox/Prox/Widgets/ReviewContainerView.swift

```diff
@@ -106,6 +115,8 @@ class ReviewContainerView: UIView {
 
         constraints += [bottomAnchor.constraint(equalTo: numberOfReviewersLabel.bottomAnchor)]
 
+        collapsedConstraints = [ heightAnchor.constraint(equalToConstant: 0) ]
```

Comment:
I'm not sure TBH, but does it matter? Don't we create a new view for every place?